indexing-service/src/main/java/io/druid/indexing/overlord/autoscaling/ec2/EC2AutoScaler.java

```diff
@@ -45,6 +46,7 @@
 public class EC2AutoScaler implements AutoScaler<EC2EnvironmentConfig>
 {
   private static final EmittingLogger log = new EmittingLogger(EC2AutoScaler.class);
+  public static final int MAX_AWS_FILTER_VALUES = 100;
```

Comment:
Is this documented by AWS somewhere? if so can we put a comment where this came from?